src/openms/extern/SQLiteCpp/CMakeLists.txt

```diff
@@ -1,22 +1,114 @@
 # Main CMake file for compiling the library itself, examples and tests.
 #
-# Copyright (c) 2012-2022 Sebastien Rombauts (sebastien.rombauts@gmail.com)
+# Copyright (c) 2012-2025 Sebastien Rombauts (sebastien.rombauts@gmail.com)
 #
 # Distributed under the MIT License (MIT) (See accompanying file LICENSE.txt
 # or copy at http://opensource.org/licenses/MIT)
-cmake_minimum_required(VERSION 3.10) # for "CMAKE_CXX_STANDARD" version
+cmake_minimum_required(VERSION 3.10)
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake") # custom CMake modules like FindSQLiteCpp
-project(SQLiteCpp VERSION 3.2.0)
+project(SQLiteCpp VERSION 3.3.3)
 
 option(SQLITECPP_BUILD_TESTS "Build and run tests." OFF)
 
+
+# BUILD_SHARED_LIBS is a standard CMake variable, but we declare it here to make it prominent
+option(BUILD_SHARED_LIBS "Build shared libraries (DLLs)." OFF)
```

Comment:
Also looks problematic since this is a global flag. Not sure what redeclaring does here.